hc-added pull request templates (#19)
Github-Community-Standards/github-community-standards: .github/PULL_REQUEST_TEMPLATE/simple_pull_request_template.md

<!-- This template should only be used in rare situations. If it is possible that the details will at all effect the ability of other contributors, the detailed pull request template should be used. If your reviewer believes more info is necessary, they may have you complete the detialed_pull_request_template. -->

## Describe your changes

## Issue ticket number and link

> 

Reviewer: ```suggestion
Resolves <ENTER THE ISSUE NUMBER> <-- You can skip if the issue is linked to the branch -->
```

Author: Ok, came up with a solution I believe.

## Checklist before requesting a review

> - [ ] I have performed a self-review of my code

Reviewer: I would hope they did but if they didn't I believe it would be obvious. Instead I would encourage people to instead make sure their Pull request is in line with the Code of Conduct and Contributing Guildlines

```suggestion
- [ ] I have reviewed the Code of Conduct
- [ ] I have reviewed the Contributing Guide
```

Author: got it makes sense, sounds better this way.

> - [ ] If it is a core feature, I have added thorough tests.

Reviewer: ```suggestion
- [ ] I have added thorough tests. <-- Skip if a documentation update -->
```

Reviewer: Tests of code in general should be required.

Author: got it!

> - [ ] Do we need to implement analytics?

Reviewer: No analytics required

```suggestion
```

Author: got it!

> - [ ] Will this be part of a product update? If yes, please write one phrase about this update.

Reviewer: All of the updates that are code won't be available without a product update. We can remove this
```suggestion
```

Author: ok, done. If u remember, I wouldn't understand a bit of clarification on this, ty.
